src/alcotest/alcotest.mli

```diff
@@ -27,26 +27,26 @@
 
     {e Release %%VERSION%%} *)
 
-include Alcotest_engine.Cli.S with type return = unit
+include Alcotest_engine.V1.Cli.S with type return = unit
 
-include module type of Alcotest_engine.Test
+include module type of Alcotest_engine.V1.Test
 (** @inline *)
 
+(** {1 Versioned APIs} *)
+
+(** An alias of the above API that provides a stability guarantees over major
+    version changes. *)
+module V1 : sig
+  include Alcotest_engine.V1.Cli.S with type return = unit
+
+  include module type of Alcotest_engine.V1.Test
+  (** @inline *)
+end
+
 (** {1 Unix-specific engine constructors}
 
     The [Alcotest_engine] package provides the most general form of the Alcotest
     API, parameterised over the thread implementation and the platform. This
     package provides the [Unix] platform implementation. *)
 
-open Alcotest_engine
-module Unix : Platform.MAKER
-
-(** {!Core.Make} is [Alcotest_engine.Core.Make (Unix)] *)
-module Core : sig
-  module Make : module type of Alcotest_engine.Core.Make (Unix)
-end
-
-(** {!Cli.Make} is [Alcotest_engine.Cli.Make (Unix)] *)
-module Cli : sig
-  module Make : module type of Alcotest_engine.Cli.Make (Unix)
-end
```

Comment:
Are `Alcotest.Core` and `Alcotest.Cli` gone? This isn't mentioned in the changelog

Reply:
Good catch, thanks; I'll add a mention of those.